docs/install_linux.md

```diff
@@ -256,20 +291,31 @@ Upon first run, the bot will automatically create the database tables needed to
 If the bot runs without any errors, you have had success so far!
 
 You can test simple functionality by running the `ping` command (With your desired prefix) in a text channel that the bot can see. If it answers "Pong!", then congratulations, *you have successfully set up WildBeast!*
+
+##RethinkDB Dashboard
+**The following information is for those who need to access the RethinkDB web interface to create, delete or edit the database contents.**
```

Comment:
I'd suggest adding `This is for users that want maximum control and are familiar with database engines, so skip this if you don't meet those criteria.` to the end of the paragraph. Just for good measure.